apps/site/src/_pages/docs/3_using-blocks.mdx

```diff
@@ -6,7 +6,10 @@ Block Protocol blocks are available in a number of embedding environments.
 
 ### HASH
 
-[HASH](https://hash.ai) is a block-based knowledge management, business intelligence and tool-building application. The hosted app is currently in private beta, and a publicly available version can be [found on GitHub](https://github.com/hashintel/hash) to self-host.
+HASH is a block-based knowledge management, business intelligence and tool-building application.
+
+- The hosted version of HASH is [available for early access](https://hash.ai) and is in alpha
+- HASH is open-source, and its code can be downloaded from the [`hashintel/hash` GitHub repository](https://github.com/hashintel/hash)
```

Comment:
```suggestion
- HASH is open source, and its code can be downloaded from the [`hashintel/hash` GitHub repository](https://github.com/hashintel/hash)
```

Reply:
This matches MS's style rules who we're free-riding off of for now, so I begrudgingly accept. Looks weird to me all the same 😅 https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

Reply:
My principle is that you hyphenate to distinguish it from things immediately following that it describes (same as MS's basically) – [Wiki](https://en.wikipedia.org/wiki/Open_source) takes the same approach. I think it's the dominant one. Probably looks weird because it's mostly encountered attached to a noun